src/pages/HomePage/Home.spec.js

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from "@testing-library/react";
+import Home from "./HomePage";
```

Comment:
does this work since your file is called `Home.js` also you are exporting `HeroSection` and not Home